templates/server-serviceaccount.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ include "dctna.serviceAccountName" . }}-server
-  namespace: {{ .Release.Namespace }}
```

Comment:
please revert this change.

Reply:
But Helm3 we manage the NameSpace outside the helm templates?  I am missing something ?

Reply:
when using `helm template` the namespaces will be in the kubectl files allowing them to be used without defining a namespace. The namespace can still be overridden using the `-n namespace`. The dicission to keep or remote these namespaces I would like to keep separate from this PR.